skynet/modules/ttt/assistant/constants.py

```diff
@@ -0,0 +1,6 @@
+assistant_default_system_message = "You are an AI assistant. You provide helpful suggestions"
+assistant_rag_question_extractor = """
+    Based on the provided text, formulate a proper question for using with RAG.
```

Comment:
Do we want to put RAG in there or something more generic like "provided documents" ?

Reply:
worked fine from my tests, let's stick to this for now imo

Reply:
Sure!